### ISSUE TITLE
Encrypted DB can be opend with wrong password

### ISSUE THREAD

**Issue by cypherkaka** [moneymanagerex/moneymanagerex#2102]
**Describe the bug**
1. Create encrypted db with password.  
2. Close the GUI and reopen
3. GUI will ask the password
4. Provide a password that set + any string
Ex. If old password is "MYPASS" then enter password "MYPASSABC"
5. It will open the GUI with wrong password "MYPASSABC"

GUI will not open if we provide a password like "ABCMYPASS".

**To Reproduce**
1. Create encrypted db with a password.  
2. Close the GUI and reopen
3. GUI will ask the password
4. Provide a password that set + any string
Ex. If old password is "MYPASS" then enter password "MYPASSABC"
5. It will open the GUI with wrong password "MYPASSABC"

**Expected behavior**
It should only open the GUI if given password is correct.

**Your OS:**
- [X] Windows
- [ ] Mac OSX
- [ ] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

**MMEX version:**
- [ ] 1.4.x
- [ ] 1.3.x
- [ ] 1.2.x
- [ ] 1.1 or older
- [ ] development version (git master branch)
MMEX is using the following support products:
wxWidgets 3.1.0
SQLite3 3.14.1
wxSQLite3 3.4.0
Mongoose 6.5
Lua 5.2
Microsoft Visual Studio 1900

**Comment by slodki** [comment]
Cannot reproduce with git master version on Linux.

**Comment by cypherkaka** [comment]
> 
> 
> Cannot reproduce with git master version on Linux.
> 
> Money Manager Ex
> Version: 1.4.0-alpha.7
> Database version: 13 (aes128cbc)
> Git commit: [ea645ba](https://github.com/moneymanagerex/moneymanagerex/commit/ea645ba838309a27796df82f4ed3b05b1c7f0ebc) (2019-03-11)
> Git branch: manual-unicode
> MMEX is using the following support products:
> • wxWidgets 3.1.2 (wxGTK 2.24)
> • wxSQLite3 4.4.2 (SQLite 3.27.2)
> • RapidJSON 1.1.0
> • Lua 5.3.3
> • libcurl 7.61.0
> • OpenSSL 1.1.1
> • zlib 1.2.11
> • libidn2 2.0.5
> • libpsl 0.20.2
> • (+libidn2 2.0.4)
> • nghttp2 1.32.1
> • librtmp 2.3
> Build on Mar 11 2019 22:39:29 with:
> • CMake 3.12.1
> • GNU Make 4.2.1
> • gettext 0.19.8.1
> • GCC 8.2.0
> • Ubuntu 18.10 'cosmic'
> 
> @cypherkaka do you have _Database version: 13 (aes128cbc)_ in About window?

@slodki ,  this bug reproduce in Windows 10 64bit.  Please see below "About" information.

Money Manager EX - Version: 1.3.3 (DB v.7)
MMEX is using the following support products:
wxWidgets 3.1.0
SQLite3 3.14.1
wxSQLite3 3.4.0
Mongoose 6.5
Lua 5.2
Microsoft Visual Studio 1900

**Comment by slodki** [comment]
It's already fixed in the next MMEX version (1.4.0).

**Comment by vomikan** [comment]
It's working for me as expected.
```
Version: 1.4.0-alpha.7
Database version: 13 (aes128cbc)
Git commit: c8f4b34f (2019-03-18)
Git branch: master
MMEX is using the following support products:
вЂў wxWidgets 3.1.2 (wxMSW 10.0)
вЂў wxSQLite3 4.4.2 (SQLite 3.27.2)
вЂў RapidJSON 1.1.0
вЂў Lua 5.3.4
вЂў libcurl 7.58.0
вЂў WinSSL
Build on Mar 18 2019 11:06:58 with:
вЂў CMake 3.9.2-MSVC_2
вЂў MSBuild 15.5.180.51428
вЂў gettext 0.19.8.1
вЂў Microsoft Visual Studio 15.0
вЂў Microsoft Visual C++ 19.12.25835.0
вЂў Windows SDK 10.0.16299.0
Running on:
вЂў Windows 10 (build 17763), 64-bit edition
вЂў Russian (windows-1251)
вЂў 1920x1080 32bit 120x120ppi 
```
